mcs_benchmark_data/cli/cli.py

```diff
@@ -39,7 +40,10 @@ def __configure_logging(self, args):
 
     def main(self):
         args = self.__parse_args()
-        self.__commands[args.command](args)
+        for command in self.__commands.values():
+            command.add_arguments(arg_parser)
+            command = self.__commands[args.command](args)

```

Comment:
Need to change __commands to have references to the command classes and not instances e.g.,
```
        self.__commands = {
            "create-benchmark-pipeline": CreateBenchmarkPipelineCommand,
            "create-submission-pipeline": CreateSubmissionPipelineCommand,
        }
```
(No parentheses, so not instantiated.)